internal/services/keyvault/key_vault_certificates_data_source_test.go

```diff
@@ -0,0 +1,83 @@
+package keyvault_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+)
+
+type KeyVaultCertificatesDataSource struct{}
+
+func TestAccDataSourceKeyVaultCertificates_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "data.azurerm_key_vault_certificates", "test")
+	r := KeyVaultCertificatesDataSource{}
+
+	data.DataSourceTest(t, []acceptance.TestStep{
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).Key("names.#").HasValue("9"),
+			),
+		},
+	})
+}
+
+func (KeyVaultCertificatesDataSource) basic(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_key_vault_certificate" "test2" {
+  count = 10
```

Comment:
we're creating 10 of these, so this would indicate that we're not paging or something?

Reply:
Paging happens after 25 resources as per doc [here](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/vendor/github.com/Azure/azure-sdk-for-go/services/keyvault/v7.1/keyvault/client.go#L2033-L2039). 
So, left untouched.

Reply:
so could we create 26 for test purposes to check we cross the boundary?